hyperhtml-v2.1.3-majinbuu-keyed/rollup.config.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+import * as path from "path";
+import babel from "rollup-plugin-babel";
+import alias from "rollup-plugin-alias";
+import resolve from "rollup-plugin-node-resolve";
+import uglify from "rollup-plugin-uglify";
+import { minify } from "uglify-es";
+
+export default {
+  input: "src/index.js",
+  output: {
+    file: "dist/index.min.js",
+    format: "iife",
+    name: "app"
+  },
+  plugins: [
+    alias({
+      'https://unpkg.com/majinbuu@latest/esm/main.js': 'node_modules/majinbuu/esm/main.js'
```

Comment:
Why is this "alias" required ?

Reply:
I can push a version that uses `rollup-plugin-cdn` but the point is that the bundler consumes hyperHTML from locally installed dependency and the external resource is only to make hyperhtml-majinbuu usable via browsers right away without needing intermediate bundlers steps.

It's a developer convenience, it doesn't affect at all this test or users of `hyperhtml-majinbuu`.

At the end, ESM is natie on browsers, and on browsers that's right now how it works.

When unpkg will be not-experimental anymore, this _could_ go away.

Since you need rollup or a package anyway, I don't see issues in the current setup.

Would you prefer `rollup-plugin-cdn` that does not need hard coded remote urls in the config?

Reply:
actually ... let me do that for you ...